src/core/congestion_control.c

```diff
@@ -27,106 +27,15 @@ QuicCongestionControlInitialize(
     CXPLAT_DBG_ASSERT(Settings->CongestionControlAlgorithm < QUIC_CONGESTION_CONTROL_ALGORITHM_MAX);
 
     switch (Settings->CongestionControlAlgorithm) {
-        case QUIC_CONGESTION_CONTROL_ALGORITHM_CUBIC: {
-            CubicCongestionControlInitialize(Cc, Settings);
-            break;
-        }
-        default: {
-            QUIC_CONNECTION* Connection = QuicCongestionControlGetConnection(Cc);
-            QuicTraceLogConnWarning(
-                InvalidCongestionControlAlgorithm,
-                Connection,
-                "Unknown congestion control algorithm: %d, fallback to Cubic",
-                Settings->CongestionControlAlgorithm);
-            CubicCongestionControlInitialize(Cc, Settings);
-            break;
-        }
+    default:
```

Comment:
This is weird. I've never seen the default case first. Just looks really odd for me. Especially with only saving 1 line.

Reply:
It's just a case statement. I've seen this way plenty of times.

Reply:
@nibanks , I don't understand this change either. What is "__fallthrough"?

Reply:
__fallthrough is just an attribute to make falling through a case statement not warn.

Reply:
Oh I see, I was confused because I didn't realize there was fallback logic and thought the code was expected to log an error trace and bail.